src/test/kotlin/no/skatteetaten/aurora/databasehotel/PostgresConnectionsExtension.kt

```diff
@@ -0,0 +1,44 @@
+package no.skatteetaten.aurora.databasehotel
+
+import no.skatteetaten.aurora.databasehotel.dao.postgres.PostgresDatabaseManager
+import org.junit.jupiter.api.extension.AfterAllCallback
+import org.junit.jupiter.api.extension.BeforeAllCallback
+import org.junit.jupiter.api.extension.ExtensionContext
+import org.springframework.test.context.junit.jupiter.SpringExtension
+import javax.sql.DataSource
+
+class PostgresConnectionsExtension : BeforeAllCallback, AfterAllCallback {
+    lateinit var databaseManager: PostgresDatabaseManager
+
+    override fun beforeAll(context: ExtensionContext?) {
+        databaseManager = context?.let { databaseManager(it) }!!
```

Comment:
Burde vi kaste noe annet enn NullPointer?

Reply:
JavaDoc for JUnit sier context aldri vil være null: https://junit.org/junit5/docs/current/api/org.junit.jupiter.api/org/junit/jupiter/api/extension/BeforeAllCallback.html (samme for AfterAllCallback)
Jeg skal endre typen på context, her og i afterAll, så den ikke er nullable

Reply:
Ja det er nok lurt :)